doc/README.md

```diff
@@ -31,3 +26,83 @@ We use a few conventions for documentation source management:
    A newline in the source has no effect on the produced content.
 2. Use two empty lines ahead of a new section.
    This keeps the source more readable.
+
+
+### A note on inline LaTeX: you can't
+
+Currently, our toolchain cannot render LaTeX snippets into PDF (works for HTML tho!).
+So we are unable to use it.
+
+What you can do though is use _cursive_, `monospace` and `.svg` images for anything more complicated.
+https://www.codecogs.com/latex/eqneditor.php is helpful for inputting LaTeX and outputting `.svg` images, or any other image format for that matter.
+We seem to use `.svg` so maybe stick to that.
+
+
+## Testing
+
+Selected parts of the source code examples in the documentation are run as tests.
+[Asciidoctor](https://github.com/asciidoctor/asciidoctor) is used to parse the documentation and extract the Python code that should be tested.
+
+
+### Installation
+
+In addition to having followed the setup steps in [the contribution guidelines](../CONTRIBUTING.md#specifically-for-this-project) and having a Neo4j database with GDS installed that can be targeted in the testing, you need to:
+
+ * Install Ruby
+ * Install `bundler`:
+   ```bash
+   gem install bundler
+   ```
+ * Install the project's Ruby dependencies (from the `doc` directory):
+   ```bash
+   bundler install tests
+   ```
```

Comment:
When following this, I encountered an error:

```
12:16 $ bundler install tests
ERROR: "bundler install" was called with arguments ["tests"]
Usage: "bundler install [OPTIONS]"
```

This was Ruby version 3.0.0

Reply:
Same error with `2.7.2`.

Reply:
These instructions do not work for me.

Reply:
```
12:22 $ bundler --version
Bundler version 2.3.21
```

Reply:
I was successful with these steps:

3. cd doc/tests
4. bundler install
5. test_docs.rb python

Reply:
:) I'll update the `bundle install` to `bundle install --gemfile tests/Gemfile`

Reply:
done